api/src/test/java/org/pmiops/workbench/api/BaseControllerTest.java

```diff
@@ -0,0 +1,68 @@
+package org.pmiops.workbench.api;
+
+import com.google.common.io.Resources;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.Random;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.runner.RunWith;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.test.FakeLongRandom;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Scope;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@DataJpaTest
+public abstract class BaseControllerTest {
```

Comment:
With a name like this, I'd expect _everything_ to extend it. Can you call this something that makes it clearer this class is intended for composition? Maybe something like `WithWorkbenchConfig`?

Reply:
Works for me. Can you coordinate with Joel to put this on the list of best practices things?